aleph-client/src/lib.rs

```diff
@@ -79,7 +109,9 @@ pub enum TxStatus {
 /// * `seed` - a 12 or 24 word seed phrase
 pub fn keypair_from_string(seed: &str) -> KeyPair {
     let pair = sr25519::Pair::from_string(seed, None).expect("Can't create pair from seed value");
-    KeyPair::new(pair)
+    KeyPair {
```

Comment:
Using constructor would be cleaner.

Reply:
You can also implement `FromStr` for KeyPair.

Reply:
done